### PR TITLE
Day 8 in go

### PR DESCRIPTION
**Pull request by tholok97**

### CODE REVIEW

File: days/day-08/solutions/tholok97.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+		if !isInfinite {
+			fmt.Println(accumulator)
+		}
+
+		instructions[i].op = swap
+	}
+}
+
+func run(instructions []instruction) (isinfinite bool, accumulator int) {
```
Comment (Arxcis) [inline days/day-08/solutions/tholok97.go:66]: Clean :100: :smiley:

File: days/day-08/solutions/tholok97.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+			instructions[i].op = "jmp"
+		default:
+			continue
+		}
+
+		isInfinite, accumulator := run(instructions)
+
+		if !isInfinite {
+			fmt.Println(accumulator)
+		}
```
Comment (Arxcis) [inline days/day-08/solutions/tholok97.go]: Break?

File: days/day-08/solutions/tholok97.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+)
+
+type instruction struct {
+	arg int
+	op  string
+}
```
Comment (Arxcis) [inline days/day-08/solutions/tholok97.go:14]: Fun!

**Review by Arxcis — APPROVED** [review]
Here are some one-word sentences that popped into my head while reviewing :smile:

File: days/day-08/solutions/tholok97.go (hunk 2 above)
Comment (tholok97) [reply days/day-08/solutions/tholok97.go]: You're right, thanks :smile: Updating

**Pull request merged**